src/style.css

```diff
@@ -0,0 +1,67 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 800;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input__search {
+  font-family: "Avenir", sans-serif;
```

Comment:
move to html selector

Reply:
moved to html but it stopped working in .input__search, I left also there this style, is it correct?